sdk/src/main/java/io/dapr/actors/communication/IActorMessageBodyFactory.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.actors.communication;
+
+/**
+ * Defines the interface that must be implemented for providing factory for
+ * creating actor request body and response body objects.
+ */
+public interface IActorMessageBodyFactory {
```

Comment:
We do not use the I* convention for interfaces. Just the name: https://azure.github.io/azure-sdk/java_design.html#naming-patterns

Reply:
No builder or factory for model classes: https://azure.github.io/azure-sdk/java_design.html#model-classes